### PR TITLE
CI: Build using LLVM 16

### PR DESCRIPTION
**Pull request by illusionyy**

### CODE REVIEW

File: .github/workflows/cmake.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        path: ${{ env.PS5SDK }}
+
+    - name: Build SDK
+      working-directory: ${{ env.PS5SDK }}
+      run: bash ./build.sh
+
+    - name: Build stubs
+      working-directory: stubber
+      run: |
+        # when stubber is fixed, we won't need this hack anymore
```
Comment (astrelsky) [inline .github/workflows/cmake.yml]: Is there still a problem?

Comment (illusionyy) [reply .github/workflows/cmake.yml]: I think the stubber still didn't ran to completion when running on Actions. I'll try it again

Comment (illusionyy) [reply .github/workflows/cmake.yml]: Mostly fixed. I had to use sudo to work around the permission issues

Comment (illusionyy) [reply .github/workflows/cmake.yml]: Removed generating stubs altogther. it uses copyrighted materials (the binaries).

**Pull request merged**